src/jobflow_remote/cli/admin.py

```diff
@@ -25,13 +25,51 @@
     loading_spinner,
     out_console,
 )
+from jobflow_remote.jobs.jobcontroller import Upgrader
 
 app_admin = JFRTyper(
     name="admin", help="Commands for administering the database", no_args_is_help=True
 )
 app.add_typer(app_admin)
 
 
+@app_admin.command()
+def upgrade(
+    force: force_opt = False,
+    test_version_upgrade: Annotated[
+        Optional[str],
+        typer.Option(
+            help="Explicitly sets the target version for upgrade. For testing purposes only",
+            hidden=True,
+        ),
+    ] = None,
+) -> None:
+    """
+    Upgrade the jobflow database.
+    WARNING: can modify all the data. Previous version could not be retrieved anymore.
+    """
+    check_stopped_runner(error=True)
+
+    jc = get_job_controller()
+    if not force:
+        jobflow_check = jc.upgrade_check_jobflow()
+        full_check = jc.upgrade_full_check()
```

Comment:
maybe the checks should go through the upgrader, rather than directly to the JobController? Or since these are just check on the db it goes directly through the Jobcontroller?